<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigEdit</class>
 <widget class="QDialog" name="ConfigEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>微服务配置</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>10</y>
     <width>691</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="savePushButton">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>配置类型名称</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="proto_nameEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>载入proto</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>载入proto</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>protobuf</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="service_nameLabel">
      <property name="text">
       <string>微服务名称</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="service_nameLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="service_ipLabel">
      <property name="text">
       <string>微服务IP</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="service_ipLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="service_portLabel">
      <property name="text">
       <string>微服务端口</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="service_portSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="proto_textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>savePushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigEdit</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigEdit</receiver>
   <slot>LoadProto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Save()</slot>
  <slot>LoadProto()</slot>
 </slots>
</ui>
